api/config/v1/sharing.go

```diff
@@ -38,7 +38,7 @@ func (s *Sharing) SharingStrategy() SharingStrategy {
 		return SharingStrategyMPS
 	}
 
-	if s.TimeSlicing.isReplicated() {
+	if s.TimeSlicing != nil && s.TimeSlicing.isReplicated() {
```

Comment:
```suggestion
	if s.TimeSlicing.isReplicated() {
```
we already return `false` if the receiver is `nil` in the `isReplicated()` implementation.

Reply:
Thanks - removed in a separate commit